ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -453,6 +453,12 @@ class SignalSpec: QuickSpec {
 				observer.sendNext(1)
 				expect(lastValue).to(equal("2"))
 			}
+
+			it("should not keep resulting signal alive indefinitely") {
+				weak var signal: Signal<AnyObject, NoError>? = Signal.never.map { $0 }
```

Comment:
I'm not sure what exactly to test there. Have both signals retained and test that the transform closure is not called after the mapped signal's observer is disposed?